src/signals/incident/definitions/wizard-step-5-samenvatting.js

```diff
@@ -26,6 +26,9 @@ export const renderPreview = ({ render: renderFunc, meta }) => {
 
       return () => 'Ja';
 
+    case 'SelectInput':
```

Comment:
You can group this with `RadioInputGroup`; uses the same format

Reply:
Indeed! Thanks.